projects/angular-formio/src/components/formbuilder/formbuilder.component.ts

```diff
@@ -154,7 +154,12 @@ export class FormBuilderComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   setDisplay(display: String) {
-    return this.builder.setDisplay(display).then(instance => this.setInstance(instance));
+    if (!this.formio?.groups?.resource) {
+      return this.builder.setDisplay(display).then(instance => this.setInstance(instance));
+    }
+    else {
+      return Promise.resolve();
```

Comment:
Something is wrong here. I do not like that we would be ignoring this call if they don't have resource groups?  Seems like a bandaid to fix a bug that is somewhere else. Please keep looking for the solution elsewhere.

Reply:
I did it another way. Please take a look